components/FormElements/formElementsStyle.ts

```diff
@@ -39,21 +39,25 @@ export const LabelStyle = styled.label`
       margin-left: 2px;
     }
   }
+
+  &:last-of-type {
+    margin-bottom: 2rem;
+  }
 `
 export const InputStyle = styled.input`
   border: none;
   background-color: ${({ theme }) => theme.bg.quarternary};
   border-bottom: 1px solid var(--mediumGrey);
   color: ${({ theme }) => theme.text.primary};
   padding: 1rem;
-  width: 70%;
+  width: 80%;
```

Comment:
I would change this at `100%` and better style the container where it's in

Reply:
I think it looks good with this on all screen sizes. ok to leave?

Reply:
In the future have it on 100%, because the that means you're already assuming that it'll be always `80%`. Maybe on some places you need it 30, 60 or 100. As you don't know it's best to add the width on the page where you need it.
For now, as it's almost presentation day, ok to leave